Source/Blazorise/Components/ModalProvider/ModalInstance.cs

```diff
@@ -0,0 +1,120 @@
+﻿#region Using directives
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+#endregion
+
+namespace Blazorise
+{
+    /// <summary>
+    /// Generates a new instance of a Modal
+    /// </summary>
+    public class ModalInstance : BaseComponent
```

Comment:
ModalInstance is only used to hold information about the modal, right? If true, then it doesn't need to inherit from BaseComponent and we don't need to have all of the BaseComponent parameters.

Reply:
Right, It was kinda of a "shortcut" to having all the base component attribute gibberish inherited. That way you don't have to manually update it every time. But since we'll have to add them manually to ModalProviderOptions... I guess we might as well do the same here.

Reply:
So added a ModalProviderBaseAttributes with all the attributes, can you take a look and remove the one's that might not make sense / do anything at all on Modal? You certainly know this better/faster then me.

Reply:
Will do.